### PR TITLE
fix(FEC-12910): allow play/pause using spacebar/enter when focused on player or seekbar

### PR DESCRIPTION
**Pull request by lianbenjamin**
### Description of the Changes

allow play/pause using spacebar/enter while focusing on seekbar component or player.

Solves FEC-12910

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Review by giladna — APPROVED** [review]

**Comment by giladna** [comment]
@lianbenjamin  please fix the flow check

File: src/components/seekbar/seekbar.js [hunk 1]
```diff
@@ -302,9 +302,24 @@ class SeekBar extends Component {
...
   };
 
+  /**
+   * toggle play/pause
+   *
+   * @returns {void}
+   * @memberof SeekBar
+   */
+  togglePlayPause = (): void => {
+    this.props.player.paused ? this.props.player.play() : this.props.player.pause();
```
Comment (SivanA-Kaltura) [inline src/components/seekbar/seekbar.js:320]: i think that the player is paused

File: src/components/shell/shell.js [hunk 2]
```diff
@@ -191,6 +192,10 @@ class Shell extends Component {
 
     if (this.state.nav && (e.keyCode === KeyMap.ENTER || e.keyCode === KeyMap.SPACE)) {
       this.unMuteFallback();
+      if (e.srcElement.contains(this._playerRef)) {
```
Comment (SivanA-Kaltura) [inline src/components/shell/shell.js:195]: why is the condition needed ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/components/shell/shell.js (hunk 2 above)
Comment (lianbenjamin) [reply src/components/shell/shell.js:195]: @SivanA-Kaltura  it seems that if you use the keyboard for other controls on the player it bubbles to the parent (the root of player), so e want to make sure we toggle play/pause when the event came from the video player element, as we don't want to do it if for example the user pressed the spacebar for seek forward button

Comment (SivanA-Kaltura) [reply src/components/shell/shell.js:195]: won't this catch every spacebar click on the root document as well though ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/components/shell/shell.js (hunk 2 above)
Comment (lianbenjamin) [reply src/components/shell/shell.js:195]: no, I checked and verified; the `event.srcElement` is different for the other controls..

**Review by semarche-kaltura — APPROVED** [review]

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**